src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/diagnostics-beginner-guide.mdx

```diff
@@ -384,32 +401,48 @@ We see that the pattern has changed and that it appears to be worsening from las
+* [Distributed tracing](/docs/distributed-tracing/concepts/introduction-distributed-tracing)
 
-We want to find the application causing the latency or errors experienced by the end-user. This doesn't mean the application code is the problem, but finding that application leads us closer to the source. You can then rule out components starting with code, database, configuration, resources, and similar.
+APM and distributed tracing are used to help you **find the source**, while logs in context can then be used to help you **debug the problem**. 
 
-Today we can use distributed tracing but first we must identify the source transaction. This means we must look at APM transactions analysis in order to identify if one, some, or all the transactions are affected by latency.
+<Callout variant="tip">
+Note that both logs in context and distributed tracing are enabled by default with our newest APM agents. (If you haven't updated your agents in a while, we recommend regularly [updating your agents](/docs/new-relic-solutions/new-relic-one/install-configure/update-new-relic-agent).)
+</Callout>
 
-Once the impacted transactions are identified, you'd search for distributed traces of that transaction. Those distributed traces will give you a more complete and holistic view of the end-to-end services within that transaction group. Distributed tracing will help you more quickly identify where the latency is or where the errors are occurring.
+First, we want to find the application causing the latency or errors experienced by the end user. This doesn't mean the application code is the problem, but finding that application leads us closer to the source. You can then rule out components starting with code, database, configuration, resources, and similar.
 
-Here's a summary of the steps described above:
+We must first identify the source transaction that's at fault. This means we must first look at APM transactions to identify if one, some, or all the transactions are affected by latency. 
 
-1. Examine applications supporting the affected capabilities using APM.
+<Callout variant="tip">
+Having well formed [service levels](/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/optimize-slm-guide) on your service boundaries relating to key transactions (capabilities) will help you more quickly identify what end-to-end workflow the problem resides in.
+</Callout>
+
+Once the impacted transactions are identified, you'd use distributed tracing to review the traces for that transaction. Those traces give you a more complete and holistic view of the end-to-end services within that transaction group, which helps you quickly identify where the latency is or where the errors are occurring.
+
+One you have found the transaction group impacted, you can use APM logs-in-context to dig down into what the logs are telling you about that problem, which will help you figure out how to resolve the incident. 
```

Comment:
I added suggestions to other mentions of "logs-in-context" (sometimes it's hyphenated, sometimes not).